doc/apidoc/smac.epm.rf_with_instances.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.rf\_with\_instances module
+=====================================
+
+.. automodule:: smac.epm.rf_with_instances
```

Comment:
@sfalkner the docstring for `rf_with_instances` mentions the parameter `max_num_nodes`. I'm currently wondering if the tree is build in depth-first or breadth-first fashion as this would change the behaviour?